### ISSUE TITLE
Updated version 2.7. preupload presentation 413 error

### ISSUE THREAD

**Issue by erpsolns** [bigbluebutton/greenlight#1973]
When uploading 
ppt 35 pages 2.1MB returns 
413 Request Entity Too Large
nginx/1.10.3 (Ubuntu)
when preuploading

pdf 35 pages 3.2MB returns
413 Request Entity Too Large
nginx/1.10.3 (Ubuntu)
when preuploading
![Screen Shot 2020-08-02 at 1 01 56 pm](https://user-images.githubusercontent.com/31876291/89114581-68c8fb00-d4c1-11ea-8c47-0462d3b69485.png)

Normal  in Meeting + Upload Presentation is working fine - no problem

**Comment by github-luckow-org** [comment]
For me adding `client_max_body_size 20M;` to the "nginx-vhost" in /etc/nginx/sites-available worked. Then it looks like
```
server {

  server_name meet.example.org;
  listen 443 ssl;
  listen [::]:443 ssl;
  client_max_body_size 20M;
```

**Comment by hex-m** [comment]
Thanks for the advise @github-luckow-org. I changed it to `30M` as that's the default size limit for BBB according to the [documentation](https://docs.bigbluebutton.org/2.2/customize.html#increase-the-file-size-for-an-uploaded-presentation).

**Comment by lkiesow** [comment]
Pull request #1977 updates the `greenlight.nginx` to allow uploads up to 25mb which should mostly fix this problem. Still, it would be nice to upload files via JavaScript and react to a 413 status to show users a sensible error instead of relying on the somewhat ugly Nginx error message where users end up outside of Greenlight.

**Comment by farhatahmad** [comment]
fixed in 2.7.1
